include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -379,32 +411,36 @@ template <typename range_type>
         }
 //!\endcond
 std::ranges::iterator_t<range_type> insert_gap(range_type & rng,
-                                               std::ranges::iterator_t<range_type> const it,
+                                               std::ranges::iterator_t<range_type> const pos_it,
                                                typename range_type::size_type const size = 1)
 {
-    return rng.insert_gap(it, size);
+    return rng.insert_gap(pos_it, size);
 }
 
 /*!\brief An implementation of seqan3::aligned_sequence::erase_gap for ranges with the corresponding
  *        member function erase_gap(it).
  * \ingroup aligned_sequence
  * \tparam range_type   Type of the range to modify; must have an erase_gap(it) member function.
  * \param[in,out] rng   The range to modify.
- * \param[in] it        The iterator pointing to the position where to erase one gap.
+ * \param[in] pos_it    The iterator pointing to the position where to erase one gap.
+ * \returns       An Iterator following the removed element. If the iterator `pos_it` refers to the last element, the
```

Comment:
:nail_care:  The spacing is weird. It is neither aligned to the others nor removed?

Reply:
What would you propose?

Reply:
I would actually remove the indentation. I was in favor of aligning everything for quite some time but it always leads to more line changes when refactoring because the indentation needs to be adapted too.

Reply:
I just changed this one instance. If you think more indentations are weird can you annotate them?